ci/build_wheel.sh

```diff
@@ -15,3 +15,6 @@ rapids-logger "Build wheel"
 export SCCACHE_S3_NO_CREDENTIALS=1
 mkdir -p ./wheel-build
 pip wheel . --wheel-dir=./wheel-build -vvv
+
+rapids-logger "Upload Wheel"
+RAPIDS_BUILD_TYPE="branch" rapids-upload-to-s3 pynvjitlink-cu12 ./wheel-build/
```

Comment:
You'll probably need to add corresponding download commands to the test_wheel.sh script.